frontend/apps/pwa/src/features/browser/RecordField.tsx

```diff
@@ -0,0 +1,42 @@
+import { FieldDefinition, Record } from 'core-api-client';
+import { FC } from 'react';
+import { Link } from 'react-router-dom';
+import format from 'date-fns/format';
+
+type RecordFieldProps = {
+  field: FieldDefinition;
+  value: any;
+  subRecords: Record[] | undefined;
+};
+export const RecordField: FC<RecordFieldProps> = (props) => {
+  const { field, value, subRecords } = props;
```

Comment:
I assume this came from copy/pasting, but we should destruct props like `= ({ field, value, subRecords }) => {